src/fileio.c

```diff
@@ -801,16 +829,167 @@ static int elf_mem_limits(const AVRMEM *mem, const AVRPART *p,
   return rv;
 }
 
+extern LISTID updates;
+
+/*  Example for AVR C code.
+For Atmega8:
+
+FUSES = { 0xC9, 0xEF };
+LOCKBITS = 0xFC;
+// Fix add .eeprom section
+#define _EEMEM  __attribute__((__used__, __section__ (".eeprom")))
+#define _EEPROM uint8_t __eeprom[512] _EEMEM
+
+_EEPROM = { 0x11, 0x22, };
+
+For Attiny4/5/9/10(Not support FUSES macros and .fuse section!!!):
```

Comment:
A little too angry perhaps. How about:
`For ATtiny4/5/9/10 (FUSES macro and .fuses section not supported by avr-libc):` ?

Reply:
I did